src/screens/DetailGroupImage/index.js

```diff
@@ -7,16 +7,16 @@ import {
   Text,
   View,
   ImageBackground,
-  StatusBar,
+  StatusBar
 } from 'react-native';
 
 import MemoIc_arrow_back_white from '../../assets/arrow/Ic_arrow_back_white';
-import {colors} from '../../utils/colors';
 import {fonts} from '../../utils/fonts';
 import Icon from 'react-native-vector-icons/Octicons';
 import {calculateTime} from '../../utils/time';
 import ModalImageSingleDetail from '../../components/Chat/ModalImageSingleDetail';
 import {useNavigation} from '@react-navigation/native';
+import {COLORS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for the colors module has been updated to use `COLORS` from the `theme` module instead of `colors`. This change is consistent with the PR's goal of standardizing color usage across the application. However, it would be beneficial to ensure that all color references in this file have been updated to use the new `COLORS` object.
```diff
- import {colors} from '../../utils/colors';
+ import {COLORS} from '../../utils/theme';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->